worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -1430,10 +1430,12 @@ public int makeSphere(Vector pos, Pattern block, double radius, boolean filled)
      * @param radiusY The sphere/ellipsoid's largest up/down extent
      * @param radiusZ The sphere/ellipsoid's largest east/west extent
      * @param filled If false, only a shell will be generated.
+     * @param hemi If true, only the top half will be generated.
+     * @param upsideDown If true and hemi is true, only the bottom half will be generated.
      * @return number of blocks changed
      * @throws MaxChangedBlocksException thrown if too many blocks are changed
      */
-    public int makeSphere(Vector pos, Pattern block, double radiusX, double radiusY, double radiusZ, boolean filled) throws MaxChangedBlocksException {
+    public int makeSphere(Vector pos, Pattern block, double radiusX, double radiusY, double radiusZ, boolean filled, boolean hemi, boolean upsideDown, @Nullable Vector directionVector) throws MaxChangedBlocksException {
```

Comment:
If you're going to add the parameter you should probably add it to the javadocs like the others :)